### ISSUE TITLE
Uzupełnić kolumny w bazie danych

### ISSUE THREAD

**Issue by NaviteLogger** [IO-Lab2/Scraper#9]
- wydawcę ( publisher ),
- rodzaj publikacji ( journal_type ),
- czasopismo ( journal ). 
- Zapakuj z punktozy dane dla publikacji dla których znajdziesz odpowiadające im czasopisma

**Comment by NaviteLogger** [comment]
@Noka993 czyli jak Kamil Ci wrzuci czasopismo, to weźmiesz mi z Punktozy rodzaj publikacji i publishera

**Comment by Noka993** [comment]
Tak. Jak czasopisma zostaną uzupełnione to dla każdej publikacji z czasopismem uzupełnię rodzaj publikacji, punkty if czasopisma i publishera. To będzie maksymalnie 38549 publikacji z 52561 wszystkich, części czasopism może nie być na punktozie

**Comment by Noka993** [comment]
@NaviteLogger Dane 4181 (maksymalna liczba dla dostępnych w bazie) publikacji zostały uzupełnione. Mogę jeszcze uzupełnić na późniejszym etapie dane do 5000 publikacji ze scrapera SGGW, których nie ma póki co w bazie, a także dane publikacji z Politechnik 
EDIT: publikacje SGGW jednak są uzupełnione, czyli zostają publikacji z Politechnik
